### DISCUSSION TITLE
Plans to release quarto for aarch64 (Apple silicon)

### DISCUSSION THREAD

**Discussion by turcotte** [quarto-dev/quarto-cli#6603] [Q&A] [answered]
### Description

Hi,

Does **Quarto** run on **Apple Silicon** (aarch64), without enabling **Rosetta**?

The installer for Quarto is prompting me to enable Rosetta, which I am trying to avoid.

I downloaded the **tar ball** and was able to execute some tasks. For instance, creating the structure for a book.

However, the **pandoc** binary in the tar ball is "**Mach-O 64-bit executable x86_64**". Therefore, my attempt to render the book as PDF failed, '**Bad CPU type in executable (os error 86)**'.

I'm surprised by this. Am I missing something? I'm new to Quarto.

Thank you for your help and time!
Marcel

**Comment by cscheid** [comment, marked as answer]
Quarto 1.3 does require Rosetta, because it uses a version of Pandoc that doesn't have Apple Silicon builds. Quarto 1.4, on the other hand, does not. If you're willing to use the pre-release versions, that's the way to go.

**Reply by turcotte** [reply]
Brilliant! Thank you for the prompt response!

Just to make the answer complete.

The package (quarto-1.4.330-macos.pkg) still requires Roesetta. However, the tarball contains both binaries, x86 and aarch64, for pandoc.

With the later, I was able to successfully render the book example to PDF on my MacBook Pro M2 Max running macOS 16.6, without enabling Rosetta.

**Reply by mpf** [reply]
Can you please say how best to install using the tarball?

**Reply by teroyks** [reply]
@mpf – follow these steps:

1. Go to the [more downloads page](https://quarto.org/docs/download/) and download the _Mac Tarball_ file.
2. Create a directory where you want to have the application (for example `~/Applications/quarto` and `cd` to that directory.
3. Unpack the file: `tar xvf ~/Downloads/{quarto-file-name}`.
  If the downloaded file name ends in `.gz` (depends on your local settings), use `tar xvzf` instead of `tar xvf`
4. You can now run quarto from the command line with `~/Applications/quarto/bin/quarto`.
